sdk/appcenter/src/test/java/com/microsoft/appcenter/utils/DeviceInfoHelperTest.java

```diff
@@ -217,6 +220,18 @@ public Void answer(InvocationOnMock invocationOnMock) {
         /* Remove wrapper SDK information. */
         DeviceInfoHelper.setWrapperSdk(null);
         assertEquals(device, DeviceInfoHelper.getDeviceInfo(contextMock));
+
+        /* Verify the right API was called to get a screen size. */
+        VerificationMode calledIn17Api = osApiLevel >= 17 ? atLeastOnce() : never();
```

Comment:
It's preferable to avoid if logic in unit tests and have distinct methods even it means duplication. Not blocking though.

Reply:
This test is already long, can we split it instead of adding new checks here?